src/app/public/pages/registration/client-registration/client-registration.component.html

```diff
@@ -93,20 +98,29 @@
               <mat-select
                 matInput
                 type="select"
-                formControlName="parish"
+                formControlName="parishID"
                 placeholder="Parish"
+                #location
                 required
               >
                 <mat-option
                   *ngFor="let parish of parishes"
-                  [value]="parish.id"
-                  >{{ parish.name }}</mat-option
+                  [value]="parish._id"
+                  >{{ parish.parish_name }}</mat-option
                 >
               </mat-select>
             </mat-form-field>
             <div>
               <button mat-button matStepperPrevious>Back</button>
-              <button mat-raised-button matStepperNext>Submit</button>
+              <button mat-raised-button matStepperNext (click)="onSubmitForm({first_name: fname.value,
```

Comment:
you can call onSubmitForm on the ts file and pull the values from the form. just a lot cleaner